src/app/screens/PaymentsHistory/styles.module.scss

```
@@ -0,0 +1,20 @@
+@import 'scss/variables/_sizes.scss';
+@import 'scss/variables/_fontSizes.scss';
+@import '_general.scss';
+@import '_colors.scss';
+
+.title {
+  font-size: $large;
+  margin-bottom: 20px;
+  margin-top: 40px;
+  text-align: center;
+}
+
+.tableContainer {
+  margin: auto;
+  width: min-content;
+}
+
+.rowHover:hover {
+  background-color: $disabled-gray !important; //No logro hacer que esto funcione
+}
```

Más allá de lo que hablamos en la meet, probá a ver si así te funciona
```suggestion
.rowHover {
&:hover {
  background-color: $disabled-gray !important;
}
```